docs/cvl/methods.md

```diff
@@ -706,6 +709,13 @@ The behavior of the `AUTO` summary depends on the type of call[^opcodes]:
   [State Mutability](https://docs.soliditylang.org/en/v0.8.12/contracts.html#state-mutability)
   in the Solidity manual for details.
 
+(assert-false-summary)=
+#### `ASSERT_FALSE` summaries
+
+This summary essentially just checks that the summarized method is not reached, 
+which can be useful in the presence of unresolved calls that can be dispatched in many ways.
+It can enable more optimizations in the prover and lead to shorter running times.
```

Comment:
Maybe?

```suggestion
This summary is a short syntax for a summary that contains an `assert false;` and essentially just checks that the summarized method is not reached.
This can be useful for instance, in the presence of unresolved calls in combination with the `unresolved external` syntax to ensure that every unresolved call is actually dispatched correctly (i.e. use `unresolved external in _._ => DISPATCH [...] default ASSERT_FALSE`). It also enables more optimizations in the prover and may lead to shorter running times.
```